doc/overview/security.md

```diff
@@ -22,21 +22,32 @@ accessing client resources or the DAOS management network.
 
 ### Client Library
 
-The client library `libdaos` is an untrusted component. The `daos` user-level 
-command that uses the client library is also an untrusted component. 
+The client library `libdaos` is an untrusted component. The `daos` user-level
+command that uses the client library is also an untrusted component.
 A trusted process, the DAOS agent (`daos_agent`),
 runs on each client node and authenticates the user processes.
 
 The DAOS security model is designed to support different authentication methods
-for client processes. Currently, we support AUTH_SYS authentication only.
+for client processes. Currently, we support only the AUTH_SYS authentication
+flavor, as defined for NFS in
+[RFC 2623](https://datatracker.ietf.org/doc/html/rfc2623#section-2.2.1).
 
 ### DAOS Management Network
 
-Each trusted DAOS component (`daos_server`, `daos_agent`, 
-and the `dmg` administrative tool) is
-authenticated by means of a certificate generated for that component. These
-components identify one another over the DAOS management network via
-mutually-authenticated TLS.
+The DAOS management components communicate over the network using the
+[gRPC protocol](https://grpc.io/).
+
+Each trusted DAOS component (`daos_server`, `daos_agent`, and the `dmg`
+administrative tool) is authenticated by means of a certificate generated for
+that component by system administrator. All of the component certificates must
+be generated with the same root certificate and distributed to the appropriate
+DAOS nodes, as described in the
+[DAOS Administration Guide](https://daos-stack.github.io/admin/deployment/#certificate-configuration).
+
+DAOS components identify one another over the DAOS management network via
+mutually-authenticated TLS over gRPC using their respective component
```

Comment:
I would swap this to read via gRPC over mutually-authenticated TLS since its more correct.

Reply:
Fixed